recipe/meta.yaml

```diff
@@ -45,7 +45,8 @@ requirements:
     - zlib
   run:
     # libwebp doesn't have run_exports yet (on version in conda-forge-pinning)
-    - libwebp
+    - libwebp       # [not (osx and arm64)]
+    - libwebp-base  # [osx and arm64]
```

Comment:
This can be helpful https://github.com/AnacondaRecipes/pillow-feedstock/blob/0505871e0d7e0f036ef8ddb2838ec5f0aca3acbd/recipe/meta.yaml#L58-L59